exe/fil-tools/cmd/fpa_add.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"os"
+	"time"
+
+	"github.com/caarlos0/spin"
+	"github.com/ipfs/go-cid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func init() {
+	fpaAddCmd.Flags().StringP("token", "t", "", "FPA access token")
+
+	fpaCmd.AddCommand(fpaAddCmd)
+}
+
+var fpaAddCmd = &cobra.Command{
+	Use:   "add [cid|path]",
+	Short: "Add data to FPA via cid or file path",
+	Long:  `Add data to FPA via a cid already in IPFS or local file path`,
+	PreRun: func(cmd *cobra.Command, args []string) {
+		err := viper.BindPFlags(cmd.Flags())
+		checkErr(err)
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second*60)
+		defer cancel()
+
+		if len(args) != 1 {
+			Fatal(errors.New("you must provide a valid cid or file path"))
```

Comment:
Nit: text error in this msg is the same as in line 73. Maybe should be diferent? (say, delete the `valid` word in this msg).